Modules/Scripted/SegmentStatistics/SegmentStatisticsCalculators/ClosedSurfaceSegmentStatisticsCalculator.py

```diff
@@ -0,0 +1,72 @@
+import vtk
+import slicer
+from SegmentStatisticsCalculators import SegmentStatisticsCalculatorBase
+
+
+class ClosedSurfaceSegmentStatisticsCalculator(SegmentStatisticsCalculatorBase):
+  """Statistical calculator for closed surfaces"""
+
+  def __init__(self):
+    super(ClosedSurfaceSegmentStatisticsCalculator,self).__init__()
+    self.name = "Closed Surface"
+    self.keys = tuple(self.name+'.'+m for m in ("surface_mm2", "volume_mm3", "volume_cc"))
```

Comment:
I simplified how keys are specified in plugins. It's now simply self.keys=["surface_mm2", "volume_mm3", "volume_cc"]. SegmentStatisticsLogic and the parameters in the ParameterNode now add the Plugin's class name+'.' as prefix to the keys where needed. Handling of this prefix is done by SegmentStatisticsLogic and SegmentsStatisticsPluginBase. Developers of plugins do not have to deal wit it.